tools/validators/instance_validator/main.py

```diff
@@ -12,7 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Main of the instance validator."""
+"""Main of the instance validator. This script takes a YAML building
+configuration file as an argument and validates it for coherence with the
+Digital Buildings ontology.
+
+This is done by first ensuring the file syntax is valid YAML, then by
+generating the ontology and comparing it with the file contents.
+
+This tool allows clients to independently validate their configuration files,
+and thus streamlines the onboarding process through automation. It saves time
```

Comment:
this part can be removed `and thus streamlines the onboarding process through automation` as it is not provided yet.

Reply:
Done